packages/solhint-plugin-mud/package.json

```diff
@@ -23,5 +23,8 @@
   "devDependencies": {
     "tsup": "^6.7.0"
   },
+  "publishConfig": {
+    "access": "public"
+  },
```

Comment:
@holic did you mean here?

Reply:
yep! though our linter will prob complain about exact placement in this, might wanna run `pnpm sort-package-json` from root

Reply:
linter seems to think this is the correct order